src/org/robockets/steamworks/Robot.java

```diff
@@ -60,8 +62,18 @@ public void robotInit() {
 		SmartDashboard.putData("Auto mode", chooser);
 		SmartDashboard.putData(new Climb(0.5));
 
+		SmartDashboard.putNumber("New Gyro Angle(AbsoluteOrRelative)", 0);
+
+		SmartDashboard.putData("GyroTurn Absolute", new Turn(TurnType.ABSOLUTE, 90)); // Angle will be on SmartDashboard from the Turn command
+		SmartDashboard.putData("GyroTurn Relative", new Turn(TurnType.RELATIVE, 90));
+
 		RobotMap.gyro.calibrate();
 
+		SmartDashboard.putNumber("GyroP", drivetrain.gyroPID.getP());
```

Comment:
Don't change anything for now, but be ready to use the SmartDashboard dumper when that's merged.